packages/app/src/app/components/UploadProgress/elements.ts

```diff
@@ -54,10 +56,12 @@ const cubeAnimation = keyframes`
 `;
 
 export const StyledCube = styled(Cube)`
-  position: absolute;
-  animation: ${cubeAnimation} 2s ease-in infinite;
-  animation-delay: ${({ i }) => i * 0.5}s;
-  transform: translateY(20px) translateX(-100px) scale(0, 0);
+  ${({ delay }: { delay: number }) => css`
```

Comment:
@Saeris I think it's better to extract the `StyledCubeProps` here, just to be consistent